<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="EEOC-2TK736001Z" ignored="0" name="Valding Part B" tests="2" failures="1" timestamp="02 Apr 2019 13:30:55 GMT" time="22.035" errors="0">
  <testcase name="partB_NegativeTest" time="2.911" classname="PartData.PartB"/>
  <testcase name="partB" time="7.584" classname="PartData.PartB">
    <failure type="java.lang.AssertionError" message="expected [Saved Part B] but found [Saved Part B.]">
      <![CDATA[java.lang.AssertionError: expected [Saved Part B] but found [Saved Part B.]
at PartData.PartB.partB(PartB.java:62)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- partB -->
</testsuite> <!-- Valding Part B -->
